deployment/.gitignore

```
@@ -1,2 +1,2 @@
-.env
-jwt.key
\ No newline at end of file
+**/.env
+**/jwt.key
```

please add line break
```suggestion
**/jwt.key

```